### PR TITLE
Add Appzi feedback form

### LINKED ISSUE #604: Add "Provide feedback" widget?
This site https://query.neurobagel.org/ has a little widget on the side for providing feedback without opening a github issue. The code addition seems relatively straightforward: https://docs.appzi.io/install . The neurobagel developer recommended it as an easy way of getting feedback from non-technical users and there is a good free tier.

### PR DESCRIPTION
**Pull request by garrettmflynn**
fix #604 

@rly @CodyCBakerPhD You're welcome to create your own account, design the form how you'd like, then swap out the token

### CODE REVIEW

**Comment by CodyCBakerPhD** [comment]
I leave this one to @rly since they pointed it out

**Comment by rly** [comment]
Thanks for the quick turnaround! I'm on vacation for the next two weeks and not able to test this, so I will review this in ~2 weeks if that's ok. (feel free to approve and merge now and I can open another issue/PR if I would like to change something.)

**Comment by garrettmflynn** [comment]
Sounds good. Have a great vacation!

**Comment by CodyCBakerPhD** [comment]
Ping @rly

**Comment by CodyCBakerPhD** [comment]
Ping @rly

**Comment by rly** [comment]
This looks great! Where do the emails go?

**Comment by garrettmflynn** [comment]
It should be going to my email and accumulated on the Appzi dashboard. Thought we'd want a shared workspace for the team—but I'm happy to take ownership of this for now. 

I can also forward to Slack if that's of interest

File: src/renderer/index.html [hunk 1]
```diff
@@ -32,5 +32,7 @@
     <nwb-dashboard></nwb-dashboard>
 
     <script type="module" src="./src/index"></script>
+    <script async src="https://w.appzi.io/w.js?token=wR0Rh"></script>
```
Comment (rly) [inline src/renderer/index.html:35]:
```suggestion
    <script async src="https://w.appzi.io/w.js?token=hUmCm"></script>
```

Changing token to the one for the NWB GUIDE account.

Comment (garrettmflynn) [reply src/renderer/index.html]: Awesome. Thanks for taking the lead on this!

**Comment by garrettmflynn** [comment]
Yes it's definitely on Appzi's end, either a call to `window.location.reload` or the equivalent default effect of the `onsubmit` callback for HTML forms.

Turns out it was more difficult than I'd imagined to block the forced reloading after Appzi submission while still allowing for QoL features like hot reloading and manual refreshing the page.

But it's working now! At this point, reloading is blocked when the Appzi form is open.

**Review by rly — APPROVED** [review]

**Comment by rly** [comment]
Thanks for the fix @garrettmflynn ! Nice work. I can confirm that the page does not refresh after submitting feedback and hot reloading and manual refreshing still work.

**Pull request merged**